pentaho-karaf-assembly/src/main/filtered-resources/etc-server/custom.properties

```diff
@@ -69,7 +69,7 @@ org.osgi.framework.system.packages.extra= \
  org.apache.log4j.*; version\="1.2.17", \
  org.apache.wml.dom; version\="2.11.0", \
  org.apache.wml; version\="2.11.0", \
- org.apache.xerces.*; version\="2.9.1", \
+ org.apache.xerces.*; version\="2.12", \
```

Comment:
You can use maven filtering here and use the variable with the correct version already defined in the parent POM.
See line 83 below for an example.

Reply:
done